go.mod

```diff
@@ -53,7 +53,6 @@ require (
 	github.com/prometheus/client_model v0.2.0 // indirect
 	github.com/prometheus/common v0.26.0 // indirect
 	github.com/prometheus/procfs v0.6.0 // indirect
-	golang.org/x/crypto v0.0.0-20211215153901-e495a2d5b3d3 // indirect
```

Comment:
The crypto pkg with the tag is required. Or the crypto with old tag is in the go.sum. it has cve issue.

If the indirect golang.org/x/crypto is removed by `go mod tidy`, may add it to the replace section

replace (
  golang.org/x/crypto => golang.org/x/crypto v0.0.0-20211215153901-e495a2d5b3d3
)

make sure the latest tag is in the go.sum after `go mod tidy`